src/util.ts

```diff
@@ -13,13 +13,39 @@ export type DecoratedNode = ESTree.Node & {
   decorators?: BabelDecorator[];
 };
 
+export type NodeWithParent = ESTree.Node & {
+  parent?: ESTree.Node;
+}
+
 /**
  * Returns if given node has a lit identifier
- * @param {ESTree.Node} node
+ * @param {NodeWithParent} node
  * @return {boolean}
  */
-function hasLitIdentifier(node: ESTree.Node): boolean {
-  return node.type === 'Identifier' && node.name === 'LitElement';
+function hasLitIdentifier(node: NodeWithParent): boolean {
```

Comment:
i wouldn't actually modify this as it was already doing the correct thing, checking if a node is a lit identifier

we should instead introduce a new function, `hasCustomElementDecorator` or something. then update `isLitClass` to look like this:

```ts
  if (hasCustomElementDecorator(klass)) {
    return true;
  }
  if (clazz.superClass) {
    return (
      hasLitIdentifier(clazz.superClass) || isLitByExpression(clazz.superClass)
    );
  }
  return hasLitIdentifier(clazz);